Address JPEG library version: library is 90, caller expects 80

blowekamp opened the pull request:
Error seen with osx 1.2.2 py3.7 package

blowekamp committed:
Remove AZP Conda installation

blowekamp committed:
Simplify OSX Conda install by using root environment

blowekamp committed:
Add testing of IO with external libraries

blowekamp committed:
Use library file path for Python configuration

blowekamp committed:
Use CMake FIND_ROOT_PATH to isolate search to conda

Disable using OSX frameworks, as CMake is finding packages in
/Library/Frameworks before in the conda PREFIX.

Configure CMake with FIND_ROOT_PATH setup to isolate search paths

blowekamp committed:
Patch SimpleITK to support more CMake framework variables

blowekamp committed:
Increment build number

zivy reviewed (approved):
If I understand correctly, the patch file is there so that the 1.2.2 release will build on conda-forge for osx and once we tag 1.2.3 we can remove it because the search paths are already updated in the release/master branches?

blowekamp commented:
That is the plan. This should work 🤞  on the PR.

blowekamp committed:
Disable explicit instantiating

blowekamp committed:
add vs2015 dependency

blowekamp committed:
Use source tar ball

blowekamp committed:
Specify SWIG 3 version

blowekamp committed:
Do not use install prefix during build

We don't wish to install the SimpleITK libraries into conda so keep
the install in the superbuild directory structure.

blowekamp commented:
🎉
